api-list.go

```diff
@@ -600,11 +641,10 @@ func (c Client) listMultipartUploadsQuery(bucketName, keyMarker, uploadIDMarker,
 	urlValues.Set("encoding-type", "url")
 
 	// maxUploads should be 1000 or less.
-	if maxUploads == 0 || maxUploads > 1000 {
-		maxUploads = 1000
+	if maxUploads > 0 {
```

Comment:
Should there be a max number?

Reply:
maximum depends on server implementation @kannappanr